src/components/TestGenerator.vue

```diff
@@ -27,6 +34,7 @@ export default {
   data() {
     return {
       numberOfTests: 1,
+      expirationDate: new Date(),
```

Comment:
I think by default we should add a few days. Just to make it easier for the person creating the tests.

Reply:
I will set it to 48 hours by default.